### ISSUE TITLE
Migrations are not running with webpack

### ISSUE THREAD

**Issue by dan003400** [PhilWaldmann/openrecord#73]
I have a Hapi app server with graphql currently using ES6 and webpack with the assistance of Backpack (https://github.com/jaredpalmer/backpack).

I can't seem to get the migrations to run at start. Any Ideas?

Thanks,

Dan

**Comment by PhilWaldmann** [comment]
Hi Dan,

I've just added some tests (see e409b0b1e203dbe6594869f1845d1cf461b55be0) and discovered a bug. If you add migrations via:

```js
const store = new Store({
  // ...
  migrations: [
    require('./path/to/one_migration'),
    require('./path/to/second_migration'),
    // ...
  ]
``` 

and your migration function does not have a name. e.g.
```js
// path/to/one_migration.js
module.exports = function(){
  this.createTable('one', function() {
    //...
  })
}
```

it wont be loaded.
I guess that's your problem.

Version 2.2.1 will create a hash based on the function code to avoid this problem. But it's always better to give it a name. I've added a note to the [docs](https://openrecord.js.org/#/migrations)

Does this solve your problem?
